### ISSUE TITLE
Is it possible for Grad-CAM to generate heatmaps for mask R-CNN?

### ISSUE THREAD

**Issue by mingqiJ** [RuoyuChen10/objectdetection-saliency-maps#5]
Hi, I see you use Faster R-CNN to do generate heatmaps for Grad-CAM. Is it possible to apply Grad-CAM to the Mask R-CNN?

I directly change the model(use PyTorch pre-trained Mask R-CNN). It didn't work.

**Comment by RuoyuChen10** [comment]
> Hi, I see you use Faster R-CNN to do generate heatmaps for Grad-CAM. Is it possible to apply Grad-CAM to the Mask R-CNN?
> 
> I directly change the model(use PyTorch pre-trained Mask R-CNN). It didn't work.

Yes, however, something may need to change. (e.g., the output, frcn has box and category, while mask rcnn has box, category and mask)

**Comment by mingqiJ** [comment]
Very thanks for your reply. I make something false before and didn't get gradient. Now fixed.
